presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -417,6 +417,30 @@ public void testShowCreateForPartitionedTablesWithDataCompression()
                         ")");
     }
 
+    @Test
+    public void testShowCreateForNonIndexedNonPartitionedTablesWithDataCompression()
```

Comment:
> NonIndexed

the test table is called indexed. is it indexed or non-indexed?

> NonPartitioned

remove

> TablesWithDataCompression

CompressedTable